src/CorsService.php

```diff
@@ -105,12 +105,6 @@ public function handlePreflightRequest(Request $request)
      */
     public function handleRequest(Request $request, Closure $next)
     {
-        try {
-            $this->validateRequest($request);
```

Comment:
1. I the `validateRequest(Request $request)` used anywhere anymore?
2. This is a BC-break

Reply:
Removed it